README.md

```diff
@@ -125,6 +125,10 @@ Emeritus Maintainers
 
 Learn more about roles in the [community repository](https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md).
 
+### No Over-Representation
+
+A community member cannot be promoted to be a Collector contrib maintainer if, after their promotion, the resulting maintainers group has more than one-fourth (25%) of the members affiliated with the same employer. In the event of confusion or concern, the OpenTelemetry Collector SIG will defer to the CNCF definition of "same employer".
```

Comment:
Sorry that I missed the original discussion, but this might need a clarification about what happens when an existing maintainer switch jobs and the new company ends up over represented. We have a clause like that for the GC, IIRC.

Reply:
This wording imposes no restriction in that case. I would be in favor of adding such restriction, but I think others differ in this

Reply:
I'm not necessarily suggesting a restriction for that case, but I think it's important to clarify what happens, so that we don't come up with rules once things happen.

I think @bogdandrutu mentioned elsewhere that it's OK to stop promoting people from specific companies when they have >= 25%, and that we might end up with over representation after specific events (job changes, maintainers stepping down), without causing further effects on the list of maintainers.

Reply:
@jpkrohling Sorry, I still don't understand what changes that would imply on this wording :sweat_smile: Could you propose the wording that you would like to see?

Reply:
Perhaps something like this? I'm happy merging the PR as is, and discuss this other improvement as a follow-up PR:

```suggestion
A community member cannot be promoted to be a Collector contrib maintainer if, after their promotion, the resulting maintainers group has more than one-fourth (25%) of the members affiliated with the same employer. Job changes and similar events might result in over-representation, and no new maintainers from the same company can be promoted until representation is balanced again. In the event of confusion or concern, the OpenTelemetry Collector SIG will defer to the CNCF definition of "same employer".
```